### PR TITLE
Services review

### PR DESCRIPTION
**Pull request by Mrt134**
- Finish updating documentation to reflect the new UI.
- Update table syntax (to be ported to master branch).
- Run make linkcheck and fix link warnings/errors (to be committed separately to master branch).
- HTML Build testing: no warnings/issues.

### CODE REVIEW

File: userguide/services.rst [hunk 1]
```diff
@@ -4,6 +4,7 @@
 Services
 ========
 
+
 The Services section of the GUI is where various services that ship
```
Comment (wblock) [inline userguide/services.rst]: "Services that ship with %brand% can be configured, started, or stopped with :guilabel:`Services`.

File: userguide/services.rst [hunk 2]
```diff
@@ -4,6 +4,7 @@
...
 with the %brand% system are configured, started, or stopped. %brand%
 includes these built-in services:
```
Comment (wblock) [inline userguide/services.rst]: The services are included:

File: userguide/services.rst [hunk 3]
```diff
@@ -134,57 +135,41 @@ which are described in
...
-   |                         |                |                                                                                                                 |
-   +-------------------------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Enable home directories | checkbox       | if checked, any user home directories located under :guilabel:`Home directories` will be available              |
-   |                         |                | over the share                                                                                                  |
-   +-------------------------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Home directories        | browse button  | select the pool or dataset which contains user home directories                                                 |
+   | Guest Account           | drop-down menu | Select an account to use for guest access. The selected account must have permissions to the pool or dataset    |
```
Comment (wblock) [inline userguide/services.rst]: s/selected//

File: userguide/services.rst [hunk 4]
```diff
@@ -134,57 +135,41 @@ which are described in
...
-   |                         |                | writable                                                                                                        |
+   | Database Path           | browse button  | Select the path to store the CNID databases used by AFP (default is the root of the pool). The path must be     |
+   |                         |                | writable.                                                                                                       |
    +-------------------------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Global auxiliary        | string         | additional `afp.conf(5) <http://netatalk.sourceforge.net/3.0/htmldocs/afp.conf.5.html>`_                        |
-   | parameters              |                | parameters not covered elsewhere in this screen                                                                 |
+   | Chmod Request           | drop-down menu | Indicates how to handle ACLs. choices are *Ignore*, *Preserve*, or *Simple*.                                    |
```
Comment (wblock) [inline userguide/services.rst]: "Set how ACLs are handled."

File: userguide/services.rst [hunk 5]
```diff
@@ -134,57 +135,41 @@ which are described in
...
    |                         |                |                                                                                                                 |
    +-------------------------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Map ACLs                | drop-down menu | choose mapping of effective permissions for authenticated users; *Rights* (default, Unix-style permissions),    |
-   |                         |                | *Mode* (ACLs), or *None*                                                                                        |
+   | Map ACLs                | drop-down menu | Choose mapping of effective permissions for authenticated users. *Rights* (default, Unix-style permissions),    |
```
Comment (wblock) [inline userguide/services.rst]: Combine sentences so choices are part of the first sentence: "users: *Rights*..."

File: userguide/services.rst [hunk 6]
```diff
@@ -207,7 +193,7 @@ problematic AFP share:
    dbd -rf /path/to/share
 
 
-This command may take a while, depending upon the size of the pool
+This command can take some time, depending upon the size of the pool
 or dataset being shared. This command will wipe the CNID database and
```
Comment (wblock) [inline userguide/services.rst]: Don't repeat "This command": "The CNID datase is wiped and rebuilt from..."

File: userguide/services.rst [hunk 7]
```diff
@@ -263,41 +250,41 @@ summarizes the available options.
...
+   | Forest Level            | drop-down menu | Choices are *2000*, *2003*, *2008*, *2008_R2*, *2012*, or *2012_R2*. Refer to                                             |
+   |                         |                | `Understanding Active Directory Domain Services (AD DS) Functional Levels                                                 |
+   |                         |                | <https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc754918(v=ws.10)>`__. |
+   |                         |                |                                                                                                                           |
+   +-------------------------+----------------+---------------------------------------------------------------------------------------------------------------------------+
+   | Administration Password | string         | Password to be used for the :ref:`Active Directory` administrator account.                                                |
```
Comment (wblock) [inline userguide/services.rst]: Should the field name change?

File: userguide/services.rst [hunk 8]
```diff
@@ -782,7 +763,8 @@ To configure any FTP scenario to use encrypted connections:
 Troubleshooting FTP
 ~~~~~~~~~~~~~~~~~~~
 
-The FTP service will not start if it cannot resolve the system's
+
+The FTP service will not start if it cannot resolve the system
 hostname to an IP address using DNS. To see if the FTP service is
```
Comment (wblock) [inline userguide/services.rst]: s/using/with/

File: userguide/services.rst [hunk 9]
```diff
@@ -793,32 +775,32 @@ running, open :ref:`Shell` and issue the command:
...
-start the FTP service, go to
-:menuselection:`System --> Advanced`,
-check the box :guilabel:`Show console messages in the footer` and
-click :guilabel:`Save`. Next, go to
-:menuselection:`Services --> Control Services`
-and switch the FTP service off, then back on. Watch the console
-messages at the bottom of the browser for errors.
+start the FTP service, go to :menuselection:`System --> Advanced`,
+check :guilabel:`Show console messages`, and click :guilabel:`Save`.
+Next, go to :guilabel:`Services` and switch the FTP service off, then
```
Comment (wblock) [inline userguide/services.rst]: s/Next, go/Go/

File: userguide/services.rst [hunk 10]
```diff
@@ -793,32 +775,32 @@ running, open :ref:`Shell` and issue the command:
...
+back on. Watch the console messages at the bottom of the browser for
+errors.
 
 If the error refers to DNS, either create an entry in the local DNS
-server with the %brand% system's hostname and IP address or add an
-entry for the IP address of the %brand% system in the
-:guilabel:`Host name database` field of
-:menuselection:`Network --> Global Configuration`.
+server with the %brand% system hostname and IP address or add an entry
```
Comment (wblock) [inline userguide/services.rst]: Comma after "address".

File: userguide/services.rst [hunk 11]
```diff
@@ -1827,33 +1787,30 @@ field include:
...
-computer and their home directory on the %brand% system, while
-restricting them from logging into the system using :command:`ssh`. To
-configure this scenario, go to
-:menuselection:`Account --> Users --> View Users`,
-select the user and click :guilabel:`Modify User`, and change the
-user's :guilabel:`Shell` to *scponly*. Repeat for each user that needs
-restricted SSH access.
+
+When SSH is configured, authenticated users with a user account
+created using :menuselection:`Account --> Users --> Add User`
```
Comment (wblock) [inline userguide/services.rst]: Put :menuselection: lines on a line by themselves.

File: userguide/services.rst [hunk 12]
```diff
@@ -1827,33 +1787,30 @@ field include:
...
+can use :command:`ssh` to login to the %brand% system over the network.
```
Comment (wblock) [inline userguide/services.rst]: s/login to/log into/

File: userguide/services.rst [hunk 13]
```diff
@@ -1827,33 +1787,30 @@ field include:
...
+The user home directory will be the pool or dataset specified in the
```
Comment (wblock) [inline userguide/services.rst]: s/will be/is/

File: userguide/services.rst [hunk 14]
```diff
@@ -1827,33 +1787,30 @@ field include:
...
+:guilabel:`Home Directory` field of their %brand% user account. While
```
Comment (wblock) [inline userguide/services.rst]: "of the %brand% account for that user."

File: userguide/services.rst [hunk 15]
```diff
@@ -1827,33 +1787,30 @@ field include:
...
+the SSH login will default to the user home directory, users are able to
```
Comment (wblock) [inline userguide/services.rst]: s/will default/defaults/

File: userguide/services.rst [hunk 16]
```diff
@@ -1994,120 +1946,122 @@ summarizes the options in the UPS Configuration screen.
...
+   |                               |                |                                                                                           |
+   +-------------------------------+----------------+-------------------------------------------------------------------------------------------+
+   | Send Email Status Updates     | checkbox       | Enables the %brand% system to send email updates to the configured field                  |
+   |                               |                | :guilabel:`To Email` address.                                                             |
+   |                               |                |                                                                                           |
+   +-------------------------------+----------------+-------------------------------------------------------------------------------------------+
+   | To Email                      | email address  | Define the email address to receive status updates.                                       |
+   |                               |                | Separate multiple email addresses with a semicolon (:kbd:`;`)                             |
```
Comment (wblock) [inline userguide/services.rst]: This should be :literal:`;`.

File: userguide/services.rst (hunk 7 above)
Comment (Mrt134) [reply userguide/services.rst]: That's what it is in the new UI - I'll open a ticket to change it back and leave a note to update the line when the ticket is resolved.

**Comment by Mrt134** [comment]
I've made the requested changes, but I'm now not allowed to push a commit to this branch for some reason. I'm going to delete the comments and try again.

**Pull request merged**